src/multilspy/language_server.py

```diff
@@ -749,12 +841,13 @@ def visit_tree_nodes_and_build_tree_repr(node: GenericDocumentSymbol) -> List[mu
     async def request_full_symbol_tree(self, within_relative_path: str | None = None, include_body: bool = False) -> List[multilspy_types.UnifiedSymbolInformation]:
         """
         Will go through all files in the project and build a tree of symbols. Note: this may be slow the first time it is called.
-        
+
         For each file, a symbol of kind Module (3) will be created. For directories, a symbol of kind Package (4) will be created.
-        All symbols will have a children attribute, thereby representing the tree structure of all symbols in the project 
+        All symbols will have a children attribute, thereby representing the tree structure of all symbols in the project
         that are within the repository.
-        Will ignore all directories that start with a dot (.) and __pycache__ directories.
-        
+        Will ignore directories starting with '.', '__pycache__', language-specific defaults (self.default_ignored_dirs),
```

Comment:
This is no longer accurate. `__pycache__` is not globally ignored afaict.

Reply:
Good catch!

Reply:
Actually, I think the defaults could be problematic in general, and it might be a good idea to make them optional (in a future update).